Packs/Vectra_AI/Integrations/VectraDetect/VectraDetect.yml

```diff
@@ -65,21 +65,31 @@ configuration:
   name: fetch_entity_types
   type: 16    # Mutiple select dropdown
   required: false
-  additionalinfo: Choose what to fetch - Hosts and/or Detections.
-    (Default - Hosts)
+  additionalinfo: Choose what to fetch - Accounts and/or Hosts and/or Detections.
+    (Default - Accounts,Hosts)
   options:
+  - Accounts
   - Hosts
   - Detections
-  defaultvalue: Hosts
+  defaultvalue: Accounts,Hosts
 
 - display: Hosts fetch query
   name: hosts_fetch_query
   type: 0    # Short text
   additionalinfo: Only "active" Hosts matching this fetch query will be fetched.
     Will be used only if "Hosts" is selected in the "Entity types to fetch".
-    (default - host.threat:>=50 AND host.certainty:>=50)
+    (default - host.threat:>=50)
   required: false
-  defaultvalue: host.threat:>=50 AND host.certainty:>=50
+  defaultvalue: host.threat:>=50
+
+- display: Accounts fetch query
```

Comment:
please add a note about this in the RN.

Reply:
Hi @YuvHayun , regarding the CommonTypes RN, I've asked the question to @kgal-pan but we were unsure how what we need to do.
I'll make the changes.
With that change, how can I put a strong dependency on the minimum CommonTypes pack version to be mandatory for my Pack version ?
Thx in advance

Reply:
Done regarding the Vectra AI Pack RN

Reply:
Actually what I meant is, that we should add the new param (accounts fetch query) to the RN.

Reply:
Oh ok, makes sense, let me do it also.

Reply:
So just making sure, everything is alright here? no more questions regarding this comment?

Reply:
Still the strong dependency question now that I've updated the CommonTypes version, is there a way to do that on my pack_metadata file ?

Reply:
Not sure I understand your questions, can you please elaborate?

Reply:
Sure, let me rephrase. As I was forced to update the "Common Types" pack to allow some of my incident type to be used with Common Types incident fields, I've added a dependency on this pack on my pack_metadata file.
However, just adding the pack name might not be sufficient if use an older version not having my changes.
So I'm asking if there is a way to say Vectra AI Pack v1.1.0 has a dependency on Common Types (minimum version 3.2.47) that way I'm sure my pack will works as expected or the mapping might failed.

Reply:
There's no need to mention that anywhere - the dependencies list is calculated automatically and will include the min version for common types.
When users will upgrade your pack it'll also upgrade Common types as well.

Reply:
Ok, I have removed the dependency as suggested